app/services/customization.ts

```diff
@@ -28,16 +30,20 @@ export class CustomizationService extends PersistentStatefulService<ICustomizati
     hideViewerCount: false,
     livedockCollapsed: true,
     previewSize: 300,
-    livedockSize: 28
+    livedockSize: 28,
+    previewEnabled: true
   };
 
+  settingsChanged = new Subject<Partial<ICustomizationSettings>>();
+
   init() {
     super.init();
     this.setLiveDockCollapsed(true);// livedock is always collapsed on app start
   }
 
   setSettings(settingsPatch: Partial<ICustomizationSettings>) {
     this.SET_SETTINGS(settingsPatch);
+    this.settingsChanged.next(settingsPatch);
```

Comment:
Does anything subscribe to this?  What is this for?

Reply:
Ups, my first attempt was event-based. Forgot to delete this.